### ISSUE TITLE
Question about heading levels

### ISSUE THREAD

**Issue by jasmussen** [WordPress/wordpress.org#382]
There are a lot of incorrect heading levels on the Make blogs, which becomes a poorly accessible experience for screen reader users. For example, these headings are h1s, they should be h2s:

![Image](https://github.com/user-attachments/assets/0ada507d-123d-4a7a-911d-14973f520abc)

That also means we need an H1 someplace on the page. That could be this one, potentially:

![Image](https://github.com/user-attachments/assets/8cb3b17d-fb85-46f7-831a-9e62b93955d2)

There are many more heading level errors, so it depends what’s feasible to do in the codebase, whether this is something that can be fixed for this first iteration or not.

**Comment by adamwoodnz** [comment]
> There are many more heading level errors, so it depends what’s feasible to do in the codebase, whether this is something that can be fixed for this first iteration or not.

These are rendered by the [o2](https://github.com/Automattic/o2) plugin, so I'm investigating what we can do there.

**Comment by adamwoodnz** [comment]
I've now fixed the heading issues on most pages; blog home and archive pages, and the new handbook layouts are done. Single post pages remain as the only major issue, but only when the welcome box is open:

| Welcome box open | Welcome box closed |
|--------|--------|
| ![Image](https://github.com/user-attachments/assets/73658725-0a8f-45e9-92e0-a930e386f557) | ![Image](https://github.com/user-attachments/assets/dbd0e5cf-4c3f-4677-b1f5-fa2c90d58a07) | 


The content of the welcome box varies across all the blogs, but they almost always contain heading tags, which come before the h1 (post title) of the page.

Do we need the welcome box on any pages except the blog home page?

cc @ndiego @fcoveram

**Comment by fcoveram** [comment]
> Do we need the welcome box on any pages except the blog home page?

I would say yes. I don't see the reason for removing them under the style update goal. Looking forward to seeing Nick's response.

**Comment by ndiego** [comment]
> Do we need the welcome box on any pages except the blog home page?

I'm not a fan of the welcome boxes, but I agree that we should leave them for this iteration and keep the functionality consistent with the existing site. Or, is there a way to force the welcome box to start closed on single pages? Not a big deal if this is complicated to implement.

**Comment by adamwoodnz** [comment]
> Or, is there a way to force the welcome box to start closed on single pages?

I'll take a look at this, but it's worth noting that once dismissed on the home page, a cookie ensures it remains closed on other pages. I always hide it, but not sure if that's what most visitors do.
